src/components/Quotes.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable */
+import React, { useState, useEffect, useRef } from "react";
+import "./Quotes.css"
+
+const Quote = () => {
+  const [quotes, setQuotes] = useState("");
+  const textRef = useRef();
+  const colors = ["#ffff00", "#90ee90", "#ffa500", "#ff68ff", "#a9a9e7"];
+  const category = "happiness";
+  const apiKey = "8vBF9RpWFlSYRkxXze3U/A==bR662D4w38AWnlA5";
+
+  const getQuotes = () => {
```

Comment:
- [ ] I highly recommend using the fetching method inside the `useEffect` hook to get it invoked on page load or component changes.

Reply:
@MahmoudBakr23 am currently using the fetch method inside the useEffect hook the above code you highlighted are just my API keys and API categories which I used inside the fetch